### PR TITLE
PDX-328: addressing release issue

### PR DESCRIPTION
**Pull request by Palak-Bansal-Provar**
seems like postinstall script is not workinga
Adding logs and synchronised mechanism to check

### CODE REVIEW

**Review by VivekSaxena — DISMISSED** [review]

File: test/commands/provar/automation/config/get.nut.ts [hunk 1]
```diff
@@ -152,13 +152,6 @@ describe('sf provar config get NUTs', () => {
     expect(getOutput.stdout).to.deep.equal('\n');
   });
 
-  it('value should be returned for testOutputLevel property', () => {
```
Comment (ajaylamba-provar) [inline test/commands/provar/automation/config/get.nut.ts:155]: Why have we done these changes? These does not seem related to the build/install issue.

Comment (Palak-Bansal-Provar) [reply test/commands/provar/automation/config/get.nut.ts:155]: as per the latest commit we are removing NUTs and values from assertion for properties that were removed from boiler plate proeprties file
for changes done under pdx-276

Comment (Palak-Bansal-Provar) [reply test/commands/provar/automation/config/get.nut.ts:155]: changes required for NUTs wrt to PDX-276

Comment (ajaylamba-provar) [reply test/commands/provar/automation/config/get.nut.ts:155]: then we should have a separate PR for that.

**Review by ajaylamba-provar — APPROVED** [review]
